### ISSUE TITLE
Quality profile does not work in stillshot mode

### ISSUE THREAD

**Issue by stanete** [afollestad/material-camera#141]
- [ ] I have verified there are no duplicate active or recent bugs, questions, or requests
- [ ] I have verified that I am using the latest version of Material Camera.
- [ ] I have given my issue a non-generic title.

###### Include the following:
 - Material Camera version: `0.4.4`
 - Device OS version: `6.0.1`
 - Device Manufacturer: `LG`
 - Device Name: `Nexus 5`

```java
new MaterialCamera(activity).saveDir(saveFolder)
                  .qualityProfile(MaterialCamera.QUALITY_480P)
                  .stillShot()
                  .start(CAMERA_IMAGE_RQ);
```

###### Expected Result

Resulting image quality should vary depending on the qualityProfile.

###### Actual Result

The image quality is always the same.

**Comment by tylerkrupicka** [comment]
Yeah stillshot doesn't take the quality in to account. We can look in to this, although I'm not sure how the quality of photos will be related to the video quality profiles and how this would effect the preview.
